asf_core_data/getters/mcs_getters/get_mcs_installations.py

```diff
@@ -111,3 +111,25 @@ def get_raw_historical_installations_data(
         ),
         dtype=base_config.raw_historical_installations_dtypes,
     )
+
+
+def get_processed_installations_data(
+    processed_installations_file_name: str,
+) -> pd.DataFrame:
+    """
+    Get a specified version of the processed MCS installation data (both domestic and non-domestic)
+    from S3 (either just MCS or MCS merged with EPC).
```

Comment:
hey @athrado ,
I created this function because @ch-williamson needed to access some older MCS processed datasets, as the current version was missing a variable he needed, for a project he is working on at the moment. Hence, instead of downloading the data from S3 and working on it, he can install the asf_core_data package and use this getter function to load a specific version of the data.

I think having "version" and "batch" as arguments is a good idea. What do you think @ch-williamson ? Since you're the one actually using the function atm.

P.S.: In the meantime, MCS have already shared the data with that specific missing variable, so when I re-run the script to re-generate processed MCS data the getter function to load the newest data can be used instead.